DigitalLearningSolutions.Data.Migrations/202307181334_updateTermsAndConditions.cs

```diff
@@ -0,0 +1,26 @@
e  (and  that  of  any  other  person  acting  on  your  authority)  of  a  Platform,  including  (without limitation) in respect of any breach of these Terms to the fullest extent permitted by law.</li><li>Without prejudice to clause 15.1, you retain responsibility for Contributions in all respects and agree to indemnify and hold harmless and continue to indemnify us and hold us harmless in all respects and in relation to all matters in respect of Contributions, including in respect of breach of the warranties given under clause 7.2, to the fullest extent permitted by law.</li></ol></li><li class=""h2 nhsuk-heading-l nhsuk-u-margin-0""><strong>SUSPENSION AND DEACTIVATION OF YOUR ACCOUNT </strong><ol class=""h4 nhsuk-body-m nhsuk-u-padding-top-4 nhsuk-u-padding-left-0""><li>We reserve the right to disable or limit the use of your Account at any time without notice and without giving any reason if you have failed to comply with any of the provisions of these Terms or if we suspect any unauthorised use or misuse of your Account, any Content or a Platform generally.   We reserve all rights not expressly granted by these Terms and conferred by law. In  the  event  that  your  Account  is  disabled,  your  right  to  access  a  Platform  shall  (without prejudice to any of our rights and remedies) terminate immediately.</li><li>Should you wish to deactivate your Account at any time, please contact us using the contact details provided in clause 2.2.  In such circumstances, we will aim to deactivate your Account within 5 (five) working days, but please note that this may take longer.</li></ol></li><li class=""h2 nhsuk-heading-l nhsuk-u-margin-0""><strong>GENERAL </strong><ol class=""h4 nhsuk-body-m nhsuk-u-padding-top-4 nhsuk-u-padding-left-0""><li><strong>Transfer of these Terms:</strong> We may transfer our rights and obligations under these Terms to another organisation, including but not limited to NHS England.   You may not transfer any of your rights or obligations under these Terms to another person.</li><li><strong>Entire agreement:</strong> These Terms and the documents expressly referred to in them contains the whole agreement between you and us relating to its subject matter and supersedes any prior agreements, representations or understandings between them unless expressly incorporated by reference in these Terms.</li><li><strong>Waiver and delay:</strong> No delay, act or omission by us in exercising any right or remedy will be deemed a waiver of that, or any other, right or remedy.</li><li><strong>Severability:</strong> If any part of these Terms is declared unenforceable or invalid, the remainder will continue to be valid and enforceable.</li><li><strong>Governing law and jurisdiction:</strong> These Terms, their subject matter and their formation (and any  non-contractual  disputes  or  claims)  are  governed  by  English  law.  We  both  agree  to  the exclusive jurisdiction of the courts of England and Wales.</li></ol></li></li></ol>'
```

Comment:
This is not the table where we need the changes. It is the 'Config' table. Please correct the query.